docs/plugins/jenkins.md

```diff
@@ -46,7 +46,19 @@ chown -R 1000:1000 <your-dtm-home-dir>/data/jenkins-volume/
 --8<-- "jenkins.yaml"
 ```
 
-Currently, all the parameters in the example above are mandatory.
+#### Default Configs
+
+| key | default value | description |
+|  ----  | ----  | ----  |
+| chart.chart_name | jenkins/jenkins | chart name |
+| chart.timeout | 10m | this config will wait 10 minutes to deploy |
```

Comment:
The default value in the yaml file is 5 minutes, but the code is set to 10 minutes.